src/sentry/api/endpoints/project_details.py

```diff
@@ -258,7 +258,8 @@ def put(self, request, project):
                         'detail': ['The new team is not found.']
                     }, status=400
                 )
-            old_team_id = project.team_id
+            # TODO(jess): update / deprecate this functionality
+            old_team_id = project.teams.values_list('id', flat=True).first()
```

Comment:
Can we change this to not use `.first()`? I'm really trying to avoid using this everywhere as a shortcut to say "give me a matching row if one exists without error".

`.first()` is _not_ a replacement for:

```python
try:
    return qs[0]
except IndexError:
    return None
```

`.first()` implies a sort order, and this imposes an extra `ORDER BY` into the query. In some cases, this additional `ORDER BY` can trigger using the wrong query plan and missing the indexes we need.